regexes/bots.yml

```diff
@@ -3849,14 +3977,41 @@
     name: 'BBC'
     url: 'https://www.bbc.com/'
 
-- regex: 'ClaudeBot'
+- regex: 'ClaudeBot|Anthropic-Claude'
   name: 'ClaudeBot'
-  category: 'Crawler'
-  url: 'https://github.com/ClaudeBot/ClaudeBot'
```

Comment:
This one technically does exist, but I guess it will be recognized as Hubot https://github.com/hubotio/hubot (which does have it's own user-agent, BTW)

Reply:
`ClaudeBot` doesn't have a user agent in the repo, so it was the Anthropic's bot. I'll open another PR for `Hubot` - https://github.com/hubotio/hubot/blob/6b9f5aa443b0f762ab2789fc2d0256be5d437d78/src/adapters/Campfire.mjs#L237